compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_desugar_phase.bal

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function addDefaultableBooleanVarsToSignature(bir:Function? func) {
+    bir:Function currentFunc = getFunction(untaint func);
```

Comment:
let's have a convention for methods like `getFunction`, how about `unwrapFunction`. https://stackoverflow.com/questions/36362020/what-is-unwrap-in-rust-and-what-is-it-used-for

@SupunS what do you think?